action.yml

```diff
@@ -23,11 +27,17 @@ runs:
   using: 'composite'
   steps:
     - run: |
+        response=$(curl --fail-with-body --silent --show-error \
+          --request POST \
+          --header "Authorization: Bearer ${{ env.SLACK_BOT_TOKEN || inputs.token }}" \
+          --header "Content-Type: application/json; charset=utf-8" \
+          --url https://slack.com/api/chat.postMessage \
+          --data "{\"channel\": \"${{ inputs.channel }}\", \"thread_ts\": \"${{ inputs.thread_ts }}\", \"unfurl_links\": ${{ inputs.unfurl_links }}, \"text\": \"${{ inputs.text }}\"}" \
+        )
+        echo "Slack API response:\n$response"
         {
-          echo "ts=<<EOF"
-          curl --silent --show-error -X POST -H "Authorization: Bearer ${{ env.SLACK_BOT_TOKEN || inputs.token }}" -H "Content-Type: application/json; charset=utf-8" --url https://slack.com/api/chat.postMessage \
-            -d "{\"channel\": \"${{ inputs.channel }}\", \"thread_ts\":\"${{ inputs.thread_ts }}\", \"text\":\"${{ inputs.text }}\"}" \
-            | jq --raw-output '.ts'
+          echo "ts<<EOF"
+          echo "$response" | jq --raw-output '.ts'
           echo "EOF"
         } >> "$GITHUB_OUTPUT"
       shell: bash
```

Comment:
It might be nice to put `shell: bash` at top to get it out of the way of this long(er) script.

```
runs:
  using: 'composite'
  steps:
    - shell:bash
      run: |
         ...
```

Reply:
https://github.com/ynab/slack-post-message-action/pull/3/commits/1f4882eb92b6ee6cc14e964c0f1e736d61a9bb2d